products/statement-generator/src/index.tsx

```diff
@@ -11,13 +12,21 @@ import './i18n';
 
 ReactDOM.render(
   <React.StrictMode>
-    <Suspense fallback={<div>Loading...</div>}>
+    <Suspense fallback={<Loading />}>
       <App />
     </Suspense>
   </React.StrictMode>,
   document.getElementById('root')
 );
 
+function Loading() {
```

Comment:
@davidwiese, lets make this its own component. I think it makes sense to put it in the `/components-layout`, but flexible to other ideas.

Reply:
Done!